pkg/validation/sessions.go

```diff
@@ -0,0 +1,32 @@
+package validation
+
+import (
+	"time"
+
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/options"
+)
+
+func validateCookieSessionMinimal(o *options.Options) []string {
```

Comment:
Perhaps `validateSessionCookieMinimal` to match the ordering elsewhere?

Reply:
Good catch.  Updated some validation output strings that I missed in switching the flag name as well.